lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -185,11 +190,11 @@ namespace {
 
        // If this op has side effects, remember it in case we need to chain it
        // to another one later.
-       if (hasSideEffects)
+       if (opHasSideEffects)
          controlDependenceValue = result;
 
        return result;
-    }
+     }
```

Comment:
Wrong indentation?

Reply:
This indentation of surrounding code is not quite correct.. I added a TODO to reformat larger chunks of code here.